component-template/{{ cookiecutter.slug }}/Makefile

```diff
@@ -0,0 +1,41 @@
+MAKEFLAGS += --warn-undefined-variables
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := all
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+DOCKER_CMD   ?= docker
+DOCKER_ARGS  ?= run -v "$${PWD}:/component" --workdir /component
+
+JSONNET_FILES   ?= $(shell find . -type f -name '*.*jsonnet' -or -name '*.libsonnet')
```

Comment:
Looks like this breaks if the files/folders contain spaces?
If I run `make` I get the following error:
```
make: *** No rule to make target `lib/{{', needed by `lint_jsonnet'.  Stop.
```